modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/layout/init.jsp

```diff
@@ -19,8 +19,7 @@ taglib uri="http://liferay.com/tld/portlet" prefix="liferay-portlet" %><%@
 taglib uri="http://liferay.com/tld/theme" prefix="liferay-theme" %><%@
 taglib uri="http://liferay.com/tld/util" prefix="liferay-util" %>
 
-<%@ page import="com.liferay.portal.kernel.portlet.PortletProvider" %><%@
-page import="com.liferay.portal.kernel.util.HtmlUtil" %><%@
+<%@ page import="com.liferay.portal.kernel.util.HtmlUtil" %><%@
```

Comment:
Is this being used?

Reply:
Jeyvison, I really not paid attention why this PortletProvider were removed. I believe it was removed after use ```gw formatSource```, but I'm not sure. 

After remove this line, everything works normally

Reply:
Should I bring it back?